### ISSUE TITLE
Error while changing Rate or Unit in Reimburse expenses page - reported by @aneequeahmad

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#10586]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Click on use icon to open setting page.
2. Click on workspace > Reimburse expenses
3. Change Track distance Rate or Unit

## Expected Result:
Should be able to change the Rate or Unit

## Actual Result:
Can't change the Rate or Unit and see error An error occurred while updating the workspace. please try again later

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.91-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y 
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/43995119/186820719-e7a23e19-9624-42ba-bd48-2d6324dc3450.mov

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @aneequeahmad
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1661445864439179

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mountiny** [comment]
Might be related to this change https://github.com/Expensify/App/pull/10374

**Comment by amyevans** [comment]
The [Web-E PR](https://github.com/Expensify/Web-Expensify/pull/34587) for the refactor of [Policy_CustomUnit_Update](https://github.com/Expensify/Expensify/issues/215182) had breaking changes in [lib/GetAPI.php](https://github.com/Expensify/Web-Expensify/pull/34587/files#diff-cb404322b33af06216873817c1583a864f2ec874002870f9571d9a07353eb919R144) to the data structure returned. For the time where Web-E changes are on prod but the [corresponding App PR](https://github.com/Expensify/App/pull/10374) is on not on prod, this is the result. Fortunately the App PR is on staging and should make it to prod shortly, so I think we're fine to just live with this issue in the meantime.

**Comment by aneequeahmad** [comment]
@madmax330 is there any bug reporting job created for this, i can see this is fixed in prod.

**Comment by mountiny** [comment]
@bfitzexpensify Would you please create reporting bonus job on upwork for @aneequeahmad and pay them out? Thank you very much!

@aneequeahmad Thanks for eagle's eye!

**Comment by bfitzexpensify** [comment]
Contract created, accepted, paid and ended

**Comment by mountiny** [comment]
🚀

**Comment by Puneet-here** [comment]
Hmm I doubt if it was eligible for the reporting bonus. There was no PR made to fix this specifically (If I am correct?) and this issue was going to get solved as mentioned above.

**Comment by aneequeahmad** [comment]
@Puneet-here, a bug which is on Production and has priority should be reported ASAP so it can be fixed on priority in a separate PR or existing PR. Does this makes sense ? Thanks

**Comment by mountiny** [comment]
@Puneet-here We discussed internally, the issue was valid at the time of reporting and it was our fault we have messed up the order of deployments and deployed frontend before the backend was ready.

We want to make sure contributors keep testing and discovering bugs and incentivize contributors to do this. The bug was valid and it was not contirbutor's fault it was fixed by backend deploy.

I agree this is an edge case which is probably not clearly described in the contributing guidelines.
